Update the "Enabling multidex support" section (#8529)
flutter/website: src/deployment/android.md

Flutter to automatically depend on `androidx.multidex:multidex` and use a
generated `FlutterMultiDexApplication` as the project's application.

When you try to build and run your app with the **Run** and **Debug** options in your IDE, your build may fail with the following message:

> <img src='/assets/images/docs/deployment/android/ide-build-failure-multidex.png' width="100%" alt='screenshot of build failure because Multidex support is required'>

Reviewer: Should this be `<img src='/assets/images/docs/android/ide-build-failure-multidex.png'` instead?

Alternatively, move the image assets to match the above code?

Author: I'll move the images to keep the paths of the links in the file consistent.

> When you try to build and run your app with the **Run** and **Debug** options in your IDE, your build may fail with the following message:
> <img src='/assets/images/docs/deployment/android/ide-build-failure-multidex.png' width="100%" alt='screenshot of build failure because Multidex support is required'>

Reviewer 2: It won't fail if you run it from the command line?

```suggestion
When you try to build and run your app with the **Run** and **Debug**
options in your IDE, your build might fail with the following message:
<img src='/assets/images/docs/deployment/android/ide-build-failure-multidex.png' width="100%" alt='screenshot of build failure because Multidex support is required'>
```

> To use the Flutter tool to enable mulidex, run `flutter run --debug` at the command line and select an Android device.
> <img src='/assets/images/docs/deployment/android/cli-select-device.png' width="100%" alt='screenshot of selecting an Android device'>
> 
> When prompted, enter `y` to add multidex support for Android. The Flutter tool will enable multidex support and retry the build.
> <img src='/assets/images/docs/deployment/android/cli-multidex-added-build.png' width="100%" alt='screenshot of a successful build after adding multidex'>
> 

Reviewer 2: ```suggestion
To enable multidex from the command line, run `flutter run --debug` and select an Android device:

<img src='/assets/images/docs/deployment/android/cli-select-device.png' width="100%" alt='screenshot of selecting an Android device'>

When prompted, enter `y`. The Flutter tool enables multidex support and retries the build:

<img src='/assets/images/docs/deployment/android/cli-multidex-added-build.png' width="100%" alt='screenshot of a successful build after adding multidex'>

```

{{site.alert.note}}
  Multidex support is natively included when targeting Android SDK 21 or later.

>   However, it isn't recommended to target API 21+ purely to resolve the multidex issue

Reviewer 2: ```suggestion
  However, we don't recommend targeting API 21+ purely to resolve the
  multidex issue.
```
